docs/geode-native-docs/function-execution/handling-function-results.html.md.erb

```diff
@@ -19,7 +19,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-Geode provides a default result collector. If you need special results handling, code a custom `ResultsCollector` implementation to replace the provided default. Use the `Execution::withCollector` method to define your custom collector.
+There is a default result collector. If you need special results handling, code a custom `ResultsCollector` implementation to replace the provided default. Use the `Execution::withCollector` method to define your custom collector.
```

Comment:
Would it be better to make the default results collector more active, e.g., "The default results collector provides the following functionality..."?